src/pages/ImportData.js

```diff
@@ -368,6 +377,16 @@ export const isProtected = (url, filetype) => {
   }
 };
 
+// This method identifies whether a workspace qualifies as protected.
+// 'Protected' here means that it has enhanced logging - either on its own or because it has an auth domain.
+// For now this also means only GCP workspaces are included.
+export const isProtectedWorkspace = (workspace) => {
```

Comment:
Do we think this will be used by other parts of Terra? If so, should it go in `workspace-utils` instead?